### PR TITLE
feat(NpmBundleSizeBadge): add new NpmBundleSizeBadge

### PR DESCRIPTION
**Pull request by tancredosouza**
This commit thus adds a new badge which uses Bundlephobia to tell
how many KB (or even MB) you will be adding with a new dependency.

This new badge is accessible as `<NpmBundleSizeBadge pkg={pkg} />`.

This pull request solves #8.

### CODE REVIEW

File: src/components/BadgesFromPkg.tsx [hunk 1]
```diff
@@ -27,6 +28,7 @@ export const badgeComponents = {
   githubIssues: GithubIssuesBadge,
   jsxReadme: JsxReadmeBadge,
   semanticRelease: SemanticReleaseBadge,
+  npmBundleSize: NpmBundleSizeBadge,
```
Comment (dbartholomae) [inline src/components/BadgesFromPkg.tsx:31]: Please move directly after the NpmDownloadsBadge

**Review by dbartholomae — CHANGES_REQUESTED** [review]
Just one small thing: Please move the new badge in BadgesFromPkg directly after the NpmDownloadsBadge

File: src/components/BadgesFromPkg.tsx (hunk 1 above)
Comment (tancredosouza) [reply src/components/BadgesFromPkg.tsx]: Sure thing! I have pushed a new commit that fixes that.

File: src/components/BadgesFromPkg.tsx [hunk 2]
```diff
@@ -40,6 +42,7 @@ export const defaultBadges: ReadonlyArray<BadgeName> = [
   "devDependenciesBadge",
   "semanticRelease",
   "jsxReadme",
+  "npmBundleSize",
```
Comment (dbartholomae) [inline src/components/BadgesFromPkg.tsx:45]: My bad - you will also have to move it in this list as this is used for sorting. Maybe you could also add a comment that this array's order determines the order in which components are displayed.

Comment (tancredosouza) [reply src/components/BadgesFromPkg.tsx]: Sorry, I didn't realize that. I've changed the code per your suggestions.

**Review by dbartholomae — APPROVED** [review]

**Comment by dbartholomae** [comment]
Thanks! Looks great! :)

**Comment by dbartholomae** [comment]
@tancredosouza I just realized that I hadn't set up CLAs for this repository yet. If you don't mind, it would be great if you could sign one for your contribution:
https://cla-assistant.io/dbartholomae/jsx-readme

Thanks! :)

**Comment by tancredosouza** [comment]
No problem! I signed it.

**Pull request merged**